ontopy/ontology.py

```diff
@@ -1207,9 +1211,18 @@ def _entities(
                 elif entity_type == "annotation_properties":
                     for prop in list(onto.annotation_properties()):
                         generator.append(prop)
+                elif entity_type == "properties":
```

Comment:
Include "properties" as a possible value to the docstring of the `entity_type` argument.